modules/rhel-images-aws.adoc

```diff
@@ -0,0 +1,29 @@
+[id="rhel-images-aws_{context}"]
+= Listing latest available RHEL images on AWS
+
+// For QE
+// Need a reason why you'd list these images in general; the note with RHEL Workers will limit this to RHEL 7. Also, where else is this used?
+Use the link:https://aws.amazon.com/cli/[AWS Command Line Interface (CLI)] to list the available RHEL images. In the AWS CLI, the following command will generate RHEL 8 Amazon Machine Images (AMI):
+
+[source, terminal]
+----
+[ec2-user@localhost ~]$ aws ec2 describe-images --owners 309956199498 <1> --query 'sort_by(Images, &CreationDate)[*].[CreationDate,Name,ImageId]' <2> --filters "Name=name,Values=RHEL-8*" <3> --region us-east-1<4> --output table<5>
+-----------------------------------------------------------------------------------------------------------
+|                                             DescribeImages                                              |
```

Comment:
We usually try to separate the command from the output so the copy button we provide has value. This is the format we've been following:

```
[source,terminal]
----
$ command
----

.Example output
[source,terminal]
----
output
----